s4e-web/projects/notifications/src/lib/components/general-notification/general-notification.component.html

```diff
@@ -0,0 +1,10 @@
+<div [ngClass]="{clickable: notification.clearable}"
+     (click)="activated.emit()"
+     class="notification-container ui-notification {{notification.type}}"
+     role="alert">
+<!--  <h3 *ngIf="notification.caption">{{notification.caption}}</h3>-->
+  <div class="message" [innerHtml]="notification.content"></div>
+<!--  <div *ngIf="notification.clearable" class="notification-action">-->
```

Comment:
Do you need those comments?

Reply:
They are for @abacz . I opted out of using caption and extended elements, but this is just my humble opinion. Maybe @abacz will decide that notifications should have captions, etc. That's why I left it here, as a hint.